### ISSUE TITLE
reporting issue List items (<li>) are not contained within <ul>, <ol> or <menu> parent elements. even though the list items are contained with <ul>

### ISSUE THREAD

**Issue by ashishsinghal77** [dequelabs/axe-core#4801]
### Product

axe-core

### Product Version

google light house version  12.6.0 

### Latest Version

- [x] I have tested the issue with the latest version of the product

#### Expectation
When the list elements are inside the `<ul>` and announcing properly, lighthouse should not report it as an issue

#### Actual
for this kind of structure 

```
<ul>
    <li>Coffee</li>
    <li>Tea</li>
    <li>Milk</li>
</ul>
```

Lighthouse is giving the following errors
List items (`<li>`) are not contained within `<ul>`, `<ol>` or <menu> parent elements.

#### How to Reproduce
Provide a code sample or link to a webpage that reproduces the issue. Without this, your issue may be closed without investigation.

#### Additional context
Any thing else we should know about the issue?

**Comment by WilcoFiers** [comment]
@ashishsinghal77 Thank you for reporting this, but the code you provided doesn't fail that rule. Can you provide a more complete example with which the issue can be reproduced?

**Comment by ashishsinghal77** [comment]
Hi Wilco,

Thanx for reaching out, here is the actual example where we are hitting the
bug

-Ashish

**Comment by ashishsinghal77** [comment]
Further here is the HTML code for same

<div id="header_menu" role="navigation">
<ul class="displayNone navMenu ui-menu ui-widget ui-widget-content" id="m2"
role="menubar" tabindex="0" style="display: block;">
<li class="ui-menu-item mainLevel"><a aria-label="Status" class="selected
ui-menu-item-wrapper" href="#" role="menuitem" aria-haspopup="true"
id="ui-id-1" tabindex="-1"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Status</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Overview" class="selected
ui-menu-item-wrapper" href="overview" id="ui-id-2" tabindex="-1"
role="menuitem">Overview</a></li>
<li class="ui-menu-item"><a aria-label="Alarms" href="alarms" id="ui-id-3"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">Alarms</a></li>
<li class="ui-menu-item"><a aria-label="System" href="#"
aria-haspopup="true" id="ui-id-4" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>System</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Information"
href="systeminformation" id="ui-id-5" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Information</a></li>
<li class="ui-menu-item"><a aria-label="Ethernet" href="ethernetstatus"
id="ui-id-6" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Ethernet</a></li>
<li class="ui-menu-item"><a aria-label="IP" href="ipstatus" id="ui-id-7"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">IP</a></li>
<li class="ui-menu-item"><a aria-label="Resource usage"
href="resourceusage" id="ui-id-8" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Resource usage</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Registrations" href="#"
aria-haspopup="true" id="ui-id-9" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Registrations</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a aria-label="By device"
href="registrations" id="ui-id-10" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">By device</a></li>
<li class="ui-menu-item"><a aria-label="By alias"
href="registrationsbyalias" id="ui-id-11" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">By alias</a></li>
<li class="ui-menu-item"><a aria-label="History" href="registrationhistory"
id="ui-id-12" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">History</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Calls" href="#"
aria-haspopup="true" id="ui-id-13" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Calls</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Calls" href="calls"
id="ui-id-14" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Calls</a></li>
<li class="ui-menu-item"><a aria-label="History" href="callhistory"
id="ui-id-15" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">History</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Search history"
href="searchhistory" id="ui-id-16" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Search history</a></li>
<li class="ui-menu-item"><a aria-label="Local Zone" href="localzonestatus"
id="ui-id-17" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Local Zone</a></li>
<li class="ui-menu-item"><a aria-label="Zones" href="zonestatus"
id="ui-id-18" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Zones</a></li>
<li class="ui-menu-item"><a aria-label="Bandwidth" href="#"
aria-haspopup="true" id="ui-id-19" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Bandwidth</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Links" href="linkstatus"
id="ui-id-20" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Links</a></li>
<li class="ui-menu-item"><a aria-label="Pipes" href="pipestatus"
id="ui-id-21" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Pipes</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Policy services"
href="policyservicestatus" id="ui-id-22" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Policy services</a></li>
<li class="ui-menu-item"><a aria-label="TURN relay usage" href="turnrelays"
id="ui-id-23" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">TURN relay usage</a></li>
<li class="ui-menu-item"><a aria-label="Unified Communications status"
href="edgestatus" id="ui-id-24" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Unified Communications status</a></li>
<li class="ui-menu-item"><a aria-label="Applications" href="#"
aria-haspopup="true" id="ui-id-25" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Applications</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a aria-label="Presence"
href="#" aria-haspopup="true" id="ui-id-26" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Presence</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Publishers"
href="publishers" id="ui-id-27" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Publishers</a></li>
<li class="ui-menu-item"><a aria-label="Presentities" href="presentities"
id="ui-id-28" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Presentities</a></li>
<li class="ui-menu-item"><a aria-label="Subscribers" href="subscribers"
id="ui-id-29" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Subscribers</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Microsoft interoperability"
href="b2buastatus" id="ui-id-30" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Microsoft interoperability</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Hardware" href="hardware"
id="ui-id-31" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Hardware</a></li>
<li class="ui-menu-item"><a aria-label="Logs" href="#" aria-haspopup="true"
id="ui-id-32" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Logs</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Event Log" href="eventlog"
id="ui-id-33" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Event Log</a></li>
<li class="ui-menu-item"><a aria-label="Configuration Log"
href="configurationlog" id="ui-id-34" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Configuration Log</a></li>
<li class="ui-menu-item"><a aria-label="Network Log" href="networklog"
id="ui-id-35" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Network Log</a></li>
</ul></li>
</ul></li>
<li class="ui-menu-item mainLevel"><a aria-label="System" href="#"
aria-haspopup="true" id="ui-id-36" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>System</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Administration settings"
href="system" id="ui-id-37" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Administration settings</a></li>
<li class="ui-menu-item"><a aria-label="Network interfaces" href="#"
aria-haspopup="true" id="ui-id-38" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Network interfaces</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a aria-label="Ethernet"
href="ethernet" id="ui-id-39" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Ethernet</a></li>
<li class="ui-menu-item"><a aria-label="IP" href="ip" id="ui-id-40"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">IP</a></li>
<li class="ui-menu-item"><a aria-label="Static routes" href="routeadd"
id="ui-id-41" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Static routes</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="DNS" href="dns" id="ui-id-42"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">DNS</a></li>
<li class="ui-menu-item"><a aria-label="Time" href="time" id="ui-id-43"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">Time</a></li>
<li class="ui-menu-item"><a aria-label="SNMP" href="snmp" id="ui-id-44"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">SNMP</a></li>
<li class="ui-menu-item"><a aria-label="Clustering" href="clustering"
id="ui-id-45" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Clustering</a></li>
<li class="ui-menu-item"><a aria-label="Protection" href="#"
aria-haspopup="true" id="ui-id-46" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Protection</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Firewall rules" href="#"
aria-haspopup="true" id="ui-id-47" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Firewall rules</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a
aria-label="Configuration" href="firewallrulesconfig" id="ui-id-48"
tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Configuration</a></li>
<li class="ui-menu-item"><a aria-label="Current active rules"
href="activefirewallrules" id="ui-id-49" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Current active rules</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Automated detection" href="#"
aria-haspopup="true" id="ui-id-50" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Automated detection</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a
aria-label="Configuration" href="protectionoverview" id="ui-id-51"
tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Configuration</a></li>
<li class="ui-menu-item"><a aria-label="Exemptions"
href="protectionexemptions" id="ui-id-52" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Exemptions</a></li>
<li class="ui-menu-item"><a aria-label="Blocked addresses"
href="protectionbanlist" id="ui-id-53" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Blocked addresses</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Rate limits" href="#"
aria-haspopup="true" id="ui-id-54" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Rate limits</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Configuration"
href="ratelimitsconfig" id="ui-id-55" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Configuration</a></li>
</ul></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Login page"
href="uploadwelcomemessage" id="ui-id-56" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Login page</a></li>
<li class="ui-menu-item"><a aria-label="Quality of Service" href="qos"
id="ui-id-57" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Quality of Service</a></li>
<li class="ui-menu-item"><a aria-label="External manager"
href="externalmanager" id="ui-id-58" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">External manager</a></li>
<li class="ui-menu-item"><a aria-label="TMS Provisioning Extension
services" href="tmsservices" id="ui-id-59" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">TMS Provisioning Extension services</a></li>
</ul></li>
<li class="ui-menu-item mainLevel"><a aria-label="Configuration" href="#"
aria-haspopup="true" id="ui-id-60" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Configuration</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a aria-label="Protocols"
href="#" aria-haspopup="true" id="ui-id-61" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Protocols</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="H.323" href="h323"
id="ui-id-62" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">H.323</a></li>
<li class="ui-menu-item"><a aria-label="SIP" href="sip" id="ui-id-63"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">SIP</a></li>
<li class="ui-menu-item"><a aria-label="Interworking" href="interworking"
id="ui-id-64" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Interworking</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Registration" href="#"
aria-haspopup="true" id="ui-id-65" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Registration</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a
aria-label="Configuration" href="registrationconfig" id="ui-id-66"
tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Configuration</a></li>
<li class="ui-menu-item"><a aria-label="Allow List" href="allowlist"
id="ui-id-67" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Allow List</a></li>
<li class="ui-menu-item"><a aria-label="Deny List" href="denylist"
id="ui-id-68" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Deny List</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Authentication" href="#"
aria-haspopup="true" id="ui-id-69" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Authentication</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a aria-label="Outbound
connection credentials" href="outboundconnectioncredentials" id="ui-id-70"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">Outbound
connection credentials</a></li>
<li class="ui-menu-item"><a aria-label="Devices" href="#"
aria-haspopup="true" id="ui-id-71" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Devices</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Local database"
href="credentials" id="ui-id-72" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Local database</a></li>
<li class="ui-menu-item"><a aria-label="Active Directory Service"
href="ntlm" id="ui-id-73" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Active Directory Service</a></li>
<li class="ui-menu-item"><a aria-label="H.350 directory service"
href="ldap" id="ui-id-74" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">H.350 directory service</a></li>
<li class="ui-menu-item"><a aria-label="H.350 directory schemas"
href="ldapschemas" id="ui-id-75" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">H.350 directory schemas</a></li>
</ul></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Call routing" href="callconfig"
id="ui-id-76" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Call routing</a></li>
<li class="ui-menu-item"><a aria-label="Local Zone" href="#"
aria-haspopup="true" id="ui-id-77" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Local Zone</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Default Subzone"
href="defaultsubzone" id="ui-id-78" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Default Subzone</a></li>
<li class="ui-menu-item"><a aria-label="Traversal Subzone"
href="traversalsubzone" id="ui-id-79" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Traversal Subzone</a></li>
<li class="ui-menu-item"><a aria-label="Subzones" href="subzones"
id="ui-id-80" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Subzones</a></li>
<li class="ui-menu-item"><a aria-label="Subzone membership rules"
href="membershiprules" id="ui-id-81" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Subzone membership rules</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Zones" href="#"
aria-haspopup="true" id="ui-id-82" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Zones</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Zones" href="zones"
id="ui-id-83" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Zones</a></li>
<li class="ui-menu-item"><a aria-label="Default Zone access rules"
href="defaultzoneaccessrules" id="ui-id-84" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Default Zone access rules</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Domains" href="domains"
id="ui-id-85" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Domains</a></li>
<li class="ui-menu-item"><a aria-label="Unified Communications" href="#"
aria-haspopup="true" id="ui-id-86" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Unified Communications</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a
aria-label="Configuration" href="edge" id="ui-id-87" tabindex="-1"
role="menuitem" class="ui-menu-item-wrapper">Configuration</a></li>
<li class="ui-menu-item"><a aria-label="Federated static routes"
href="staticroutes" id="ui-id-88" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Federated static routes</a></li>
<li class="ui-menu-item"><a aria-label="Federated domains allow list"
href="federationallowlist" id="ui-id-89" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Federated domains allow list</a></li>
<li class="ui-menu-item"><a aria-label="Federated domains deny list"
href="federationdenylist" id="ui-id-90" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Federated domains deny list</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Dial plan" href="#"
aria-haspopup="true" id="ui-id-91" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Dial plan</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Configuration"
href="dialplanconfig" id="ui-id-92" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Configuration</a></li>
<li class="ui-menu-item"><a aria-label="Transforms" href="transforms"
id="ui-id-93" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Transforms</a></li>
<li class="ui-menu-item"><a aria-label="Search rules" href="searchrules"
id="ui-id-94" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Search rules</a></li>
<li class="ui-menu-item"><a aria-label="Policy services"
href="policyservices" id="ui-id-95" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Policy services</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Bandwidth" href="#"
aria-haspopup="true" id="ui-id-96" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Bandwidth</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Configuration"
href="bandwidth" id="ui-id-97" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Configuration</a></li>
<li class="ui-menu-item"><a aria-label="Links" href="links" id="ui-id-98"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">Links</a></li>
<li class="ui-menu-item"><a aria-label="Pipes" href="pipes" id="ui-id-99"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">Pipes</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Traversal" href="#"
aria-haspopup="true" id="ui-id-100" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Traversal</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Ports" href="ports"
id="ui-id-101" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Ports</a></li>
<li class="ui-menu-item"><a aria-label="TURN" href="turn" id="ui-id-102"
tabindex="-1" role="menuitem" class="ui-menu-item-wrapper">TURN</a></li>
<li class="ui-menu-item"><a aria-label="Locally registered endpoints"
href="locallyregisteredendpoints" id="ui-id-103" tabindex="-1"
role="menuitem" class="ui-menu-item-wrapper">Locally registered
endpoints</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Call Policy" href="#"
aria-haspopup="true" id="ui-id-104" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Call Policy</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Configuration"
href="adminpolicy" id="ui-id-105" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Configuration</a></li>
<li class="ui-menu-item"><a aria-label="Rules" href="callpolicywizard"
id="ui-id-106" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Rules</a></li>
</ul></li>
</ul></li>
<li class="ui-menu-item mainLevel"><a aria-label="Applications" href="#"
aria-haspopup="true" id="ui-id-107" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Applications</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a aria-label="Presence"
href="presence" id="ui-id-108" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Presence</a></li>
<li class="ui-menu-item"><a aria-label="B2BUA" href="#"
aria-haspopup="true" id="ui-id-109" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>B2BUA</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Microsoft interoperability"
href="#" aria-haspopup="true" id="ui-id-110" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Microsoft interoperability</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a
aria-label="Configuration" href="b2buaconfig" id="ui-id-111" tabindex="-1"
role="menuitem" class="ui-menu-item-wrapper">Configuration</a></li>
<li class="ui-menu-item"><a aria-label="Trusted hosts"
href="b2bualisteners" id="ui-id-112" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Trusted hosts</a></li>
<li class="ui-menu-item"><a aria-label="Restart service..."
href="b2buarestart" id="ui-id-113" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Restart service...</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="B2BUA TURN servers"
href="b2buaturn" id="ui-id-114" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">B2BUA TURN servers</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="FindMe" href="userpolicy"
id="ui-id-115" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">FindMe</a></li>
<li class="ui-menu-item"><a aria-label="Cloud Services" href="#"
aria-haspopup="true" id="ui-id-116" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Cloud Services</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a aria-label="Connector
Management" href="cloudregistration" id="ui-id-117" tabindex="-1"
role="menuitem" class="ui-menu-item-wrapper">Connector Management</a></li>
<li class="ui-menu-item"><a aria-label="Connector Proxy" href="cloudproxy"
id="ui-id-118" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Connector Proxy</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Cloud Certificate management"
href="fusioncerts" id="ui-id-119" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Cloud Certificate management</a></li>
</ul></li>
<li class="ui-menu-item mainLevel"><a aria-label="Users" href="#"
aria-haspopup="true" id="ui-id-120" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Users</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Password security"
href="adminpasswordsecurity" id="ui-id-121" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Password security</a></li>
<li class="ui-menu-item"><a aria-label="Forbidden password dictionary"
href="editpassworddictionary" id="ui-id-122" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Forbidden password dictionary</a></li>
<li class="ui-menu-item"><a aria-label="Administrator accounts"
href="adminaccounts" id="ui-id-123" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Administrator accounts</a></li>
<li class="ui-menu-item"><a aria-label="Administrator groups"
href="admingroups" id="ui-id-124" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Administrator groups</a></li>
<li class="ui-menu-item"><a aria-label="Active administrator sessions"
href="adminsessions" id="ui-id-125" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Active administrator sessions</a></li>
<li class="ui-menu-item"><a aria-label="LDAP configuration"
href="loginldap" id="ui-id-126" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">LDAP configuration</a></li>
</ul></li>
<li class="ui-menu-item mainLevel"><a aria-label="Maintenance" href="#"
aria-haspopup="true" id="ui-id-127" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Maintenance</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Upgrade" href="upgrade"
id="ui-id-128" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Upgrade</a></li>
<li class="ui-menu-item"><a aria-label="Logging" href="logging"
id="ui-id-129" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Logging</a></li>
<li class="ui-menu-item"><a aria-label="Smart licensing"
href="smartlicensing" id="ui-id-130" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Smart licensing</a></li>
<li class="ui-menu-item"><a aria-label="Email Notifications"
href="emailnotifications" id="ui-id-131" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Email Notifications</a></li>
<li class="ui-menu-item"><a aria-label="Tools" href="#"
aria-haspopup="true" id="ui-id-132" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Tools</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Locate" href="locate"
id="ui-id-133" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Locate</a></li>
<li class="ui-menu-item"><a aria-label="Check pattern" href="checkpattern"
id="ui-id-134" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Check pattern</a></li>
<li class="ui-menu-item"><a aria-label="Generate passphrase"
href="generatepassphrase" id="ui-id-135" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Generate passphrase</a></li>
<li class="ui-menu-item"><a aria-label="Port usage" href="#"
aria-haspopup="true" id="ui-id-136" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Port usage</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Local inbound ports"
href="localportlist" id="ui-id-137" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Local inbound ports</a></li>
<li class="ui-menu-item"><a aria-label="Local outbound ports"
href="sourceportlist" id="ui-id-138" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Local outbound ports</a></li>
<li class="ui-menu-item"><a aria-label="Remote listening ports"
href="remoteportlist" id="ui-id-139" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Remote listening ports</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Network utilities" href="#"
aria-haspopup="true" id="ui-id-140" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Network utilities</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a aria-label="Ping"
href="ping" id="ui-id-141" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Ping</a></li>
<li class="ui-menu-item"><a aria-label="Traceroute" href="traceroute"
id="ui-id-142" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Traceroute</a></li>
<li class="ui-menu-item"><a aria-label="Tracepath" href="tracepath"
id="ui-id-143" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Tracepath</a></li>
<li class="ui-menu-item"><a aria-label="DNS lookup" href="dnslookup"
id="ui-id-144" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">DNS lookup</a></li>
<li class="ui-menu-item"><a aria-label="Connectivity test"
href="connectivitytest" id="ui-id-145" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Connectivity test</a></li>
</ul></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Security" href="#"
aria-haspopup="true" id="ui-id-146" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Security</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Trusted CA certificate"
href="trustedcacertificate" id="ui-id-147" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Trusted CA certificate</a></li>
<li class="ui-menu-item"><a aria-label="Server certificate"
href="servercertificate" id="ui-id-148" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Server certificate</a></li>
<li class="ui-menu-item"><a aria-label="CRL management" href="crlupdater"
id="ui-id-149" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">CRL management</a></li>
<li class="ui-menu-item"><a aria-label="Client certificate testing"
href="certificatetesting" id="ui-id-150" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Client certificate testing</a></li>
<li class="ui-menu-item"><a aria-label="Certificate-based authentication
configuration" href="cbaconfig" id="ui-id-151" tabindex="-1"
role="menuitem" class="ui-menu-item-wrapper">Certificate-based
authentication configuration</a></li>
<li class="ui-menu-item"><a aria-label="Domain certificates"
href="domaincertificate" id="ui-id-152" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Domain certificates</a></li>
<li class="ui-menu-item"><a aria-label="Ciphers" href="ciphers"
id="ui-id-153" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Ciphers</a></li>
<li class="ui-menu-item"><a aria-label="SSH configuration" href="sshconfig"
id="ui-id-154" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">SSH configuration</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Backup and restore"
href="backuprestore" id="ui-id-155" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Backup and restore</a></li>
<li class="ui-menu-item"><a aria-label="Diagnostics" href="#"
aria-haspopup="true" id="ui-id-156" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Diagnostics</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Diagnostic logging"
href="loggingsnapshot" id="ui-id-157" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Diagnostic logging</a></li>
<li class="ui-menu-item"><a aria-label="System snapshot" href="snapshot"
id="ui-id-158" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">System snapshot</a></li>
<li class="ui-menu-item"><a aria-label="Incident reporting" href="#"
aria-haspopup="true" id="ui-id-159" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Incident reporting</a><ul role="menu"
aria-hidden="true" class="ui-menu ui-widget ui-widget-content ui-front"
style="display: none;"><li class="ui-menu-item"><a
aria-label="Configuration" href="incidentreporting" id="ui-id-160"
tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Configuration</a></li>
<li class="ui-menu-item"><a aria-label="View" href="incidentview"
id="ui-id-161" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">View</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Advanced" href="#"
aria-haspopup="true" id="ui-id-162" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper"><span class="ui-menu-icon ui-icon
ui-icon-caret-1-e"></span>Advanced</a><ul role="menu" aria-hidden="true"
class="ui-menu ui-widget ui-widget-content ui-front" style="display:
none;"><li class="ui-menu-item"><a aria-label="Network Log configuration"
href="networkloglevels" id="ui-id-163" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Network Log configuration</a></li>
<li class="ui-menu-item"><a aria-label="Support Log configuration"
href="developerloglevels" id="ui-id-164" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Support Log configuration</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Hybrid Services Log Levels"
href="hybridservicesloglevels" id="ui-id-165" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Hybrid Services Log Levels</a></li>
</ul></li>
<li class="ui-menu-item"><a aria-label="Maintenance mode"
href="maintenancemode" id="ui-id-166" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Maintenance mode</a></li>
<li class="ui-menu-item"><a aria-label="Language" href="webprefs"
id="ui-id-167" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Language</a></li>
<li class="ui-menu-item"><a aria-label="Restart options"
href="restartoptions" id="ui-id-168" tabindex="-1" role="menuitem"
class="ui-menu-item-wrapper">Restart options</a></li>
</ul></li>
</ul></div>

**Comment by ashishsinghal77** [comment]
Ok, thanks for the clarification.

-Ashish
